challenges/deploy-WebV/README.md

```diff
@@ -0,0 +1,78 @@
+# Deploy Web Validate
+
+## Background
+
+Web Validate (WebV) is an end-to-end test tool that we will use to generate web requests against our application. These web requests can be validated on a simple returned status code or can be configured to perform a deep validation. Crucially, the logs and metrics of the testing can be stored in Graphana or Azure Logs to ensure application functionality and performance through integration and load testing. Although not part of this challenge, dashboards and monitors are created from these logs and metrics to gain a real time view of the application along with its historical baseline.
+
+More information about running WebV interactively from the command line can be found [here](https://github.com/microsoft/webvalidate).
+
+The WebV application needs to be deployed on our AKS cluster. Create a Kubernetes manifest file (yaml) to define and create the appropriate resource(s) needed to deploy WebV. The inputs needed to deploy WebV on our AKS cluster are shown below. Using these inputs, WebV will generate approximately 1 request per second when deployed.
+
+```yaml
+
+namespace:
+    ngsa
+args:
+    --sleep 1000 --run-loop --server https://${ASB_TEAM_NAME}.aks-sb.com --files memory-benchmark.json
+
+```
+
+### Prerequisite
+
```

Comment:
we import ngsa-app in the challenge

they will need to import webv

Reply:
done